src/code/rand.c

```diff
@@ -42,6 +42,7 @@
  * @note Original name: qrand.c
  */
 #include "ultra64.h"
+#include "rand.h"
```

Comment:
```suggestion
#include "rand.h"
```
`rand.h` includes `ultra64.h`